Modules/AbRouter/Http/Controllers/StatisticsController.php

```diff
@@ -66,10 +66,12 @@ public function showStats(
             $request->input('filter.date_from'),
             $request->input('filter.date_to')
         ));
-        
+
         return [
             'percentage' => $results->getPercentage(),
             'counters' => $results->getCounters(),
```

Comment:
Should be like: 
array_merge($results->getCounters(), $results->getRevenueCounters())

Revenue is the wrong term. It's about the use case. Not a feature.